### PR TITLE
Fix Spanish translations acording to README.md

### PR DESCRIPTION
**Pull request by ccmorataya**
Update about #199 and #237

### CODE REVIEW

**Comment by kitao** [comment]
It looks the top link to Pyxel logo doesn't work. How did you check the updated README?

File: README.es.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# <img src="https://raw.githubusercontent.com/kitao/pyxel/master/images/pyxel_logo_152x64.png">
+# <img src="https://raw.githubusercontent.com/kitao/pyxel/master/pyxel/images/pyxel_logo_152x64.png">
```
Comment (kitao) [inline README.es.md:1]: This link is invalid.

File: README.es.md [hunk 2]
```diff
@@ -58,25 +58,25 @@ Pyxel es de código abierto y de libre uso. Comencemos a crear juegos retro con
 
 Primero, instalar [Python3](https://www.python.org/) (versión 3.6.8 o superior).
 
-Cuando instale Python con el instalador oficial, **añada Python al PATH** marcando el botón de abajo:
+Cuando instales Python con el instalador oficial, **añadir Python al PATH** marcando la casilla de abajo:
 
-<img src="https://raw.githubusercontent.com/kitao/pyxel/master/images/python_installer.png">
+<img src="https://raw.githubusercontent.com/kitao/pyxel/master/pyxel/images/python_installer.png">
```
Comment (kitao) [inline README.es.md:63]: This link is invalid. Please check the latest REAMDE.md

**Comment by ccmorataya** [comment]
Sorry, I've fixed that link @kitao

File: README.es.md [hunk 3]
```diff
@@ -344,26 +344,26 @@ Dibuja la pantalla y espera indefinidamente (no usarlo en aplicaciones normales)
 Guarda el archivo de recurso (.pyxres) en el directorio donde se ejecuta el script
```
Comment (kitao) [inline README.es.md:344]: sound and music mean the name of Pyxel instances (class) and can be obtained with the same name APIs. So it should not be translated.

**Comment by kitao** [comment]
@ccmorataya 
I'm very sorry to left this PR. Finally I managed to check PRs again.
And I've fixed the conflicts between the links of the latest README and this pull requests.

I'm eager to merge your updated translation, on the other hand, several class names such as Sound and Music are translated and it can cause some confusion because users can't use the translated class name.

So could you modify your translation to distinguish the class names and explanation of them?

**Comment by kitao** [comment]
There seems to be no progress in this case, so I will close it once.

**Pull request closed without merging**